sceneview_1_0_0/src/main/java/io/github/sceneview/nodes/ModelNode.kt

```diff
@@ -151,6 +151,8 @@ open class ModelNode private constructor(
      * @see animationCount
      */
     fun playAnimation(animationIndex: Int = 0, loop: Boolean = true) {
+        if (animationCount == 0) return
+
         if (animationIndex <= animationCount) {
```

Comment:
By the way, the inequality should be strict here since the last animation index is one less than the animation count. This will also help to eliminate the crash.

Reply:
Updated thanks!